README.rst

```diff
@@ -61,6 +61,7 @@ CLI Options
       num-task:                     number of task nodes
       release-label:                EMR release label
       s3-bucket:                    name of s3 bucket to upload spark file (required)
+      s3-path:                      path within s3-bucket to use when writing assets (optional)
```

Comment:
Not: don't thin we need to call out optional. I think we only explicitly call out when required.